newsfragments/41536.significant.rst

```diff
@@ -0,0 +1,15 @@
+**Breaking Change**
+
+The ``get_conn_uri`` method has been removed from the ``AzureKeyVaultBackend`` class.
```

Comment:
What is the motivation to do this breaking change?

Also providers are not part of core cycle so if we want to so this we need entry in the provider change log not a newsfragemnt (we dont have yet newsfragment for peoviders)

Reply:
Ah i see your correct, will remove this update this PR :) thank you

Reply:
was trying to to remove airflow3 deprecation warnings in BaseSecretsBackend , thought of first removing in from the usage places. Please suggest should i ignore doing these changes in providers?